_includes/head.html

```diff
@@ -9,6 +9,7 @@
     <meta name="keywords" content="{{ page.tags | join: ', ' }}" />
     {% endif %}
     {% if page.date %}
+    <meta name="robots" content="nofollow">
```

Comment:
please put the "<meta name="robots" content="nofollow">" out of the {% if... %}{% endif %} block